README.md

```diff
@@ -8,10 +8,12 @@ The main functions of Workflow is to efficiently parallelise operations over a s
 
 For examples and more information see [documentation](https://libatoms.github.io/workflow/)
 
-NOTE: because of the very large time intervals between official ASE releases, `wfl` is typically
-set up for (and tested against) the latest ASE gitlab repo `master` branch.  Recent changes
-that require this support include variable cell minimization using `FrechetCellFilter` and
-`Espresso` calculator configuration. See documentation link above for installation instructions.
+`wfl` its dependendensies may be installed vi `pip install wfl`. 
+
+NOTE: `wfl` is only tested against the latest ASE pip release, currently v3.23. 
+For the time being, ASEv3.22 is allowed with the most recent branch of `wfl`, hoewever,
+not fully supported (e.g. the correct file-based calculator keywords must be specified
```

Comment:
I don't think this is really an example of "not fully supported". It's got less automatic managing of those keywords, but it's perfectly happy to pass the args and/or env var to the ASE calculator.

I'm going to edit these sentences,

Reply:
If you are happy with the little wording changes I put it, please post here, and I'll merge.  If you think I mis-stated the current compatibility status, we should fix that first.

Reply:
That's clearer, thanks. 

(I've just fixed minor typos in "`wfl` its dependendensies may be installed vi `pip install wfl`. ")